shop.html

```diff
@@ -263,7 +263,7 @@ <h2 class="card-heading">Madhubani Art</h2>
             ></i>
           </div>
         </div>
-        <div class="card simple-card reset margin-md">
+        <div class="card card-simple reset margin-md">
           <img src="./assets/item3.jfif" alt="" class="card-img" />
```

Comment:
check the shop.html file, there are a few alt tags missing

Reply:
> check the shop.html file, there are a few alt tags missing

Thank you for pointing those things out. Did all the suggested changes